async-to-promises.test.js

```diff
@@ -238,7 +240,16 @@ for (const name of fs.readdirSync("tests").sort()) {
 					const { babel, parse, types, pluginUnderTest, pluginMapping, checkOutput } = environments[
 						babelName
 					];
-					const mappedPlugins = plugins.map((pluginName) => pluginMapping[pluginName]);
+					const mappedPlugins = plugins.map((pluginName) => {
+						const mappedPlugin = pluginMapping[pluginName];
+						if (mappedPlugin) {
+							return mappedPlugin;
+						}
+
+						throw new Error(
```

Comment:
It wasn't totally clear to me at first that the plugins specified in test cases were meant to reference some identifier that would then get mapped to the Babel version-specific plugin name. I think this makes it a bit more clear if you pass an actual plugin name.